spec/selectors/item_spec.rb

```diff
@@ -22,7 +22,7 @@
 
   it "does not select an missing type" do
     expect do
-      expect(page).to have_selector :item, "age", type: "foo", text: "222"
```

Comment:
Not something that's urgent but at some point wait will likely need to be set to `0` I imagine. If/when we get more strict on input validations

Reply:
Oh are they planning to do that? I assumed false was equivalent to 0.

Reply:
It is yes. However I think if/when there is any work to optimise stuff passed into selenium, this might be something to consider. 

Moreover it's a "maybe", but there is a translation layer. So yeh it's no biggie. Just an FYI if anything.